potatohead.html

```diff
@@ -15,7 +15,7 @@
 			<img id="body" src="assets/body.png">
 			<img id="arms_image">
 			<img id="ears_image">
-			<img id="eyes_image" >
+			<img id="eyes.png" >
```

Comment:
This doesn't look quite right. Try again, I'm sure you'll get it next time!

Be sure to try it out in Cloud9 before committing.